addons/Rakugo/Rakugo.gd

```diff
@@ -103,7 +103,8 @@ func _ready():
 	var title = ProjectSettings.get_setting(Rakugo.game_title)
 	OS.set_window_title(title + " " + version)
 	
-	define_character("narrator", Rakugo.narrator_name)
+	var narrator_name = ProjectSettings.get_setting(Rakugo.narrator_name)
+	define_character("narrator", narrator_name)
```

Comment:
You can do
```gd
define_character("narrator", ProjectSettings.get_setting(Rakugo.narrator_name))
```
without creating a var if you want :)

Reply:
I know, I just don't like long lines.
But you right there is no need for this extra var, so I will change it to one line.

Reply:
If you don't like long line you can do that
```gdscript
define_character("narrator",
  ProjectSettings.get_setting(Rakugo.narrator_name)
)
```
or
```gdscript
define_character(
  "narrator",
  ProjectSettings.get_setting(Rakugo.narrator_name)
)
```
As you want :)